docs/victorialogs/vlagent.md

```diff
@@ -0,0 +1,510 @@
+---
+weight: 3
+menu:
+  docs:
+    parent: victorialogs
+    weight: 3
+title: vlagent
+tags:
+  - logs
+aliases:
+  - /vlagent.html
+  - /vlagent/index.html
+  - /vlagent/
+---
+
+`vlagent` is a tiny agent which helps you collect logs from various sources
+and store them in [VictoriaLogs](https://docs.victoriametrics.com/victorialogs/).
+See [Quick Start](#quick-start) for details.
+
+
+## Motivation
+
+While VictoriaLogs provides an efficient solution to store and observe logs, it lacks of replication out of box.
+Previous solution was to configure clients to replicate log streams into multiple VictoriaLogs installations.
+`vlagent` is a missing piece of log streams replication.
+
+## Features
+
+- It can accept logs from popular log collectors. See [these docs](https://docs.victoriametrics.com/victorialogs/data-ingestion/).
+* Can replicate collected metrics simultaneously to multiple VictoriaLogs instances - see [these docs](#replication-and-high-availability).
```

Comment:
```suggestion
* Can replicate collected logs simultaneously to multiple VictoriaLogs instances - see [these docs](#replication-and-high-availability).
```

Reply:
I noticed that "metrics" is used instead of "logs" in some parts of the doc. When you have a moment, could you please review the text?